ch5/Makefile

```
@@ -19,5 +19,6 @@ server:
 	docker run --name api-server -p 8000:8000 -d ch5-api-server
 
 server-clean:
-	docker rm -f api-server
-	docker rmi ch5-api-server
+	echo "------------Cleaning Chapter 5 resources------------"
+	docker rm -f api-server || true
+	docker rmi ch5-api-server || true
```

```suggestion
	docker rm -f api-server
	docker rmi -f ch5-api-server
```
혹시 안 고치신 이유가 있을까요? 아니면 commit 이 누락된 것 같습니다!

---

#24 에서 마무리 되었습니다!